pylint/checkers/imports.py

```diff
@@ -490,6 +490,8 @@ def _record_import(self, node, importedmodnode):
                 importedname = node.modname
             else:
                 importedname = node.names[0][0].split('.')[0]
+        if node.as_string().startswith('from .'):
```

Comment:
Thanks for feedback
I added the sentence `if isinstance(node, astroid.ImportFrom)`
But I can't use `modname` insteado of `node.as_string().startswith('from .')``

I added the following diff
```diff
diff --git a/pylint/checkers/imports.py b/pylint/checkers/imports.py
index 27a181d..e098376 100644
--- a/pylint/checkers/imports.py
+++ b/pylint/checkers/imports.py
@@ -490,8 +490,11 @@ given file (report RP0402 must not be disabled)'}
                 importedname = node.modname
             else:
                 importedname = node.names[0][0].split('.')[0]
-        if node.as_string().startswith('from .'):
-            importedname = '.' + importedname
+        if isinstance(node, astroid.ImportFrom) and 'wrong_import_order.py' in node.root().file:
+            print "node.modname", [node.modname],
+            print "importedname", [importedname], node.root().file + ':' + str(node.lineno)
+            if node.as_string().startswith('from .'):
+                importedname = '.' + importedname
         self._imports_stack.append((node, importedname))

     @staticmethod
```

I saw the following output:
```bash
coverage run -m unittest discover -s pylint/test -p test_functional.py
...ss.s....s....s...s...s........s.........s...s.s.........s.....................s..ss......ss...s.................s...sss.....sss..........ss........s.s.s............s....s.s....s.s.....s.....s...s.....node.modname ['six.moves'] importedname ['six.moves'] ./pylint/pylint/test/functional/wrong_import_order.py:4
node.modname ['astroid'] importedname ['astroid'] ./pylint/pylint/test/functional/wrong_import_order.py:10
node.modname ['package'] importedname ['package'] ./pylint/pylint/test/functional/wrong_import_order.py:14
node.modname [''] importedname ['functional'] ./pylint/pylint/test/functional/wrong_import_order.py:16
node.modname [''] importedname ['functional'] ./pylint/pylint/test/functional/wrong_import_order.py:18
node.modname ['package2'] importedname ['package2'] ./pylint/pylint/test/functional/wrong_import_order.py:19
...sss.
----------------------------------------------------------------------
Ran 210 tests in 10.746s

OK (skipped=41)
```

Could you help me to know how get the `.` (first point) from `modname`?